Assets/Scripts/UI/MainMenu/OptionsMenu.cs

```diff
@@ -25,36 +26,87 @@ public class OptionsMenu : MonoBehaviour
     #endregion
     #region Audio Variables
     [Header("Audio")]
-    // Exposed Audiomixer variable names
-    private const string audioGroupMaster = "masterVolume";
-    private const string audioGroupMusic = "musicVolume";
-    private const string audioGroupSFX = "sfxVolume";
 
-    private float maxVolumeMusic;
-    private float maxVolumeSFX;
+    [SerializeField]
+    private Slider masterVolumeSlider;
+    [SerializeField]
+    private Slider musicVolumeSlider;
+    [SerializeField]
+    private Slider sfxVolumeSlider;
 
+    #endregion
+    #region Sensitivity variables
+    [Header("Sens Elements")]
     [SerializeField]
-    private AudioMixer mainAudioMixer;
+    private Slider sensitivitySlider;
+    [SerializeField]
+    private TMP_InputField sensitivityInputField;
+
+    [Header("FOV Elements")]
+    [SerializeField]
+    private TMP_InputField FOVInputField;
+    [SerializeField]
+    private TMP_InputField ZoomFOVInputField;
+    [SerializeField]
+    private Slider FOVSlider;
+    [SerializeField]
+    private Slider ZoomFOVSlider;
 
     #endregion
 
-    void Awake()
+    [SerializeField]
+    private SettingsInfo settingsInfo;
+
+    private void Start()
     {
         CheckResolutions();
         CheckQuality();
-        mainAudioMixer.GetFloat(audioGroupMusic, out float musicVolume);
-        maxVolumeMusic = musicVolume;
-        mainAudioMixer.GetFloat(audioGroupSFX, out float sfxVolume);
-        maxVolumeSFX = sfxVolume;
+
+        settingsInfo = SettingsInfo.Singleton;
+        sensitivitySlider.minValue = settingsInfo.lowerSensLimit;
+        sensitivitySlider.maxValue = settingsInfo.upperSensLimit;
+
+        FOVSlider.minValue = settingsInfo.lowerFOVLimit;
+        FOVSlider.maxValue = settingsInfo.upperFOVLimit;
+
+        ZoomFOVSlider.minValue = settingsInfo.lowerZoomFOVLimit;
+        ZoomFOVSlider.maxValue = settingsInfo.upperZoomFOVLimit;
+
+        SetOptionItems();
     }
 
+    private void SetOptionItems()
+    {
+        resolutionDropdown.value = settingsInfo.settings.resolutionPresetIndex;
+
+        fullscreenTypeDropdown.value = settingsInfo.settings.resolutionPresetIndex;
+
+        qualityDropdown.value = settingsInfo.settings.qualityPresetIndex;
+
+        masterVolumeSlider.value = settingsInfo.settings.masterVolume;
+
+        musicVolumeSlider.value = settingsInfo.settings.musicVolume;
+
+        sfxVolumeSlider.value = settingsInfo.settings.sfxVolume;
+
+        sensitivitySlider.value = settingsInfo.settings.sensScale;
+        sensitivityInputField.text = settingsInfo.settings.sensScale.ToString("0.00");
+
+        FOVSlider.value = settingsInfo.settings.playerFOV;
+        FOVInputField.text = settingsInfo.settings.playerFOV.ToString("0.00");
+
+        ZoomFOVSlider.value = settingsInfo.settings.zoomFOV;
+        ZoomFOVInputField.text = settingsInfo.settings.zoomFOV.ToString("0.00");
```

Comment:
Could we format the FOV values as whole numbers, please? The .xx is a little distracting when it likely doesn't matter much.